dll/win32/syssetup/wizard.c

```diff
@@ -604,6 +742,11 @@ ProductPageDlgProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam)
                     {
                         OnChooseOption(hwndDlg, pSetupData->ProductOption);
                         DoWriteProductOption(pSetupData->ProductOption);
+                        if (PRODUCT_OPTION_DEFAULT == PRODUCT_OPTION_WORKSTATION)
+                        {
+                            /* Write system sound events values for Workstation */
+                            DoWriteSoundEvent();
+                        }
```

Comment:
This entire block of code is executed only in the case of unattended setup, your function call should be actually inside `DoWriteProductOption()`

Reply:
OK, I moved it there, but will it still work properly only for Workstation, if I just call it directly? If no, then how I can check for Workstation option? :thinking: